addons/mail/static/src/models/attachment_image/attachment_image.js

```diff
@@ -61,7 +61,7 @@ registerModel({
             if (this.attachmentList.composerView) {
                 return 50;
             }
-            if (this.attachmentList.chatter) {
+            if (this.attachmentList.attachmentBoxViewOwner && this.attachmentList.attachmentBoxViewOwner.chatter) {
```

Comment:
Still have probably unnecessary guard on chatter here?

Reply:
Look like this one can go too